server/game/core/Player.js

```diff
@@ -882,6 +882,16 @@ class Player extends GameObject {
         }
     }
 
+    /**
+     * Ready the specified number of resources
+     */
+    readyResources(count) {
```

Comment:
Need logic to handle the case when count > number of available resources. My thought is to just add a contract check for that and make it illegal to do instead of making it allowed, I think allowing it as an option would create too many potential errors.

Reply:
What kind of errors do you think this could cause? This function is just a slight alteration of exhaustResources() right above it, which has been fine. count being higher than the number of exhausted resources is handled by the Math.min() in in for loop declaration.